lib/msal-core/src/utils/CryptoUtils.ts

```diff
@@ -50,13 +50,13 @@ export class CryptoUtils {
             buffer[8] &= 0xbf; // buffer[8] & 10111111 will set the 6 bit to 0.
 
             return CryptoUtils.decimalToHex(buffer[0]) + CryptoUtils.decimalToHex(buffer[1])
```

Comment:
Can this not be done with a loop or iterator?

Reply:
Probably, but this code is copy/pasta, and only part of this diff due to the whitespace fixes below.

Reply:
Oh I blanked on the copy/paste, mb.

Reply:
Is there a reason why it is not done in the firstplace?

Reply:
Not sure, I'm guessing it was copied from somewhere else. In any case, not in scope for this PR.